hack/verify-doctoc.sh

```diff
@@ -22,7 +22,10 @@ if [[ "${TRACE-0}" == "1" ]]; then
     set -o xtrace
 fi
 
-command -v doctoc || echo "doctoc is not available on your system, skipping verification" && exit 0
+if [[ -z "$(command -v doctoc)" ]]; then
+  echo "doctoc is not available on your system, skipping verification"
+  exit 0
+fi
 
 doctoc_files="README.md \
```

Comment:
The list of files here seems pretty limited - but we don't need tables of contents everywhere. should we at least add it to proposals? Not sure how else we'd like to enforce this in the book, but it's worked fine until now.

Reply:
Will do

Reply:
I'll open an issue and mark it as help for this.

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/9114